src/index.js

```diff
@@ -109,15 +130,9 @@ function syncReduxAndRouter(history, store, selectRouterState = SELECT_STATE) {
   const unsubscribeStore = store.subscribe(() => {
     let routing = getRouterState();
 
-    // Treat `firstRoute` as our `initialState`
-    if(routing === initialState) {
-      routing = firstRoute;
-    }
-
     // Only trigger history update is this is a new change or the
     // location has changed.
```

Comment:
Tiny typo: "Only trigger history update is" should be "Only trigger history update if"